capa/v25.0.0/README.md

```diff
@@ -0,0 +1,157 @@
+# :zap: Giant Swarm Release v25.0.0 for CAPA :zap:
+
+We are happy to announce our first `Cluster API for AWS` (CAPA) release v25. 
+
+This is the first Giant Swarm supported CAPA release. It is available on CAPA Management Clusters and will be used as a first release to be upgraded to from `Vintage` workload clusters.
+
+## CAPA benefits
+
+Each existing customer using Giant Swarm Vintage product has been given a presentation about CAPA benefits. We have gathered the most crucial high-level advantages in the list below to refresh your memory:
+
+- Easier cluster management, complete production ready clusters packaged as a Helm chart, more GitOps oriented approach
+- Better visibility of subresources, clear API
+- More transparent upgrades
+- Improved cluster configuration validation
+- Management clusters capable of deploying clusters in different regions
+- Flexible network configuration
+- Automatic certificate renewal in place for worker nodes
+- Exposing more features based on the upstream implementations and contributions
+- [Karpenter](https://karpenter.sh/) solution integrated into workload clusters out of the box
+- More configurable registries with credentials and mirrors
+
+## Important changes
+
+Besides the benefits listed above, we have also presented changes that are introduced with CAPA. Here is the summary of most important points:
+
+- Flat DNS structure - Bringing more flexibility into workload cluster management across regions. Old and new DNS are both available during and after migration, while new DNS needs to be used after migration. We will synchronize with customers for the old DNS phase out after the migrations per management cluster.
+- Only public DNS zone is supported - Private hosted zones are no longer created. However, both zones (public and private) will be available during and after the migration. We will eventually work together to remove the private zones with each customer. Ingresses that depend on the cluster base domain must be updated after the migration.
+- Cluster apps moved to `org-` namespaces - This change will allow to simplify RBAC and enable GitOps managed clusters to be created with a pre-defined set of applications.
+- One set of controllers - On contrary to Vintage, where each release had a set of independent operators, the CAPA solution will manage all releases with single set of controllers. This will result in smaller set of components running on the management clusters, hence lowering costs of operations as well as speeding up any required hotfixes.
```

Comment:
```suggestion
- One set of controllers - On contrary to Vintage, where each release had a set of independent operators, the CAPA solution will manage all releases with a single set of controllers. This will result in fewer components running on the management clusters, hence lowering costs of operations as well as speeding up any required hotfixes.
```